src/main/java/org/irods/operations/UserGroupOperations.java

```diff
@@ -106,6 +197,12 @@ public Response removeFromGroup(String token, String user, String zone, String g
         return new Response(response.statusCode(), response.body());
     }
 
+    /**
+     * Lists all users in the zone. This operation requires rodsadmin level privileges.
```

Comment:
does this require admin?  hmmm.

Reply:
The API documentation said it required admin so I just copied and pasted what that said.

Reply:
Created #98 for this.